doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Installation/business-central-system-properties-ref.adoc

```diff
@@ -1,78 +1,91 @@
 +
-[NOTE]
-====
-If you plan to use RSA or any algorithm other than DSA, make sure you set up your application server to use the Bouncy Castle JCE library.
-====
-* `org.uberfire.metadata.index.dir`: Place where the Lucene `.index` directory is stored. Default: the working directory
-* `org.uberfire.ldap.regex.role_mapper`: Regex pattern used to map LDAP principal names to the application role name. Note that the variable role must be part of the pattern because it is substituted by the application role name when matching a principal value to a role name. Default: Not used.
-* `org.uberfire.sys.repo.monitor.disabled`: Disables the configuration monitor. Do not disable unless you are sure. Default value: `false`
-* `org.uberfire.secure.key`: Password used by password encryption. Default value: `org.uberfire.admin`
-* `org.uberfire.secure.alg`: Crypto algorithm used by password encryption. Default value: `PBEWithMD5AndDES`
-* `org.uberfire.domain`:  Security-domain name used by uberfire. Default value: `ApplicationRealm`
-* `org.guvnor.m2repo.dir`: Place where the Maven repository folder is stored. Default value: `<working-directory>/repositories/kie`
-* `org.guvnor.project.gav.check.disabled`: Disables group ID, artifact ID, and version (GAV) checks. Default value: `false`
-* `org.kie.build.disable-project-explorer`: Disables automatic build of a selected project in Project Explorer. Default value: `false`
-* `org.kie.verification.disable-dtable-realtime-verification`: Disables the real-time validation and verification of decision tables. Default value: `false`
-* `org.kie.server.controller`: URL for connecting with a {CONTROLLER}, for example: `ws://localhost:8080/{URL_COMPONENT_CENTRAL}/websocket/controller`
+NOTE: If you plan to use RSA or any algorithm other than DSA, make sure you set up your application server to use the Bouncy Castle JCE library.
+
+* `org.uberfire.metadata.index.dir`: Place where the Lucene `.index` directory is stored. Default value: Working directory.
+* `org.uberfire.ldap.regex.role_mapper`: Regex pattern used to map LDAP principal names to the application role name. Note that the variable role must be part of the pattern because it is substituted by the application role name when matching a principal value to a role name.
+* `org.uberfire.sys.repo.monitor.disabled`: Disables the configuration monitor. Do not disable unless you are sure. Default value: `false`.
+* `org.uberfire.secure.key`: Password used by password encryption. Default value: `org.uberfire.admin`.
+* `org.uberfire.secure.alg`: Crypto algorithm used by password encryption. Default value: `PBEWithMD5AndDES`.
+* `org.uberfire.domain`:  Security-domain name used by uberfire. Default value: `ApplicationRealm`.
```

Comment:
There is a double space before "Security-domain".

Reply:
Good catch!